apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,394 @@
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/contact-us-title")}
+        </Typography>
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/contact-us-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4">
+        <div className="flex flex-row gap-2">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/email/1")}
+          </Typography>
+          <Typography
+            tag="h3"
+            weight="medium"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/email/2")}
+          </Typography>
+        </div>
+
+        <div className="flex flex-row gap-2">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/phone/1")}
+          </Typography>
+          <Typography
+            tag="h3"
+            weight="medium"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/phone/2")}
+          </Typography>
+        </div>
+
+        <div className="flex flex-col sm:flex-row flex-wrap gap-4">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/social-media")}
+          </Typography>
+
+          <div className="flex gap-2">
+            <a
+              href="/instagram"
+              className="flex items-center justify-center bg-natural-50 h-10 w-10 rounded-full"
+            >
+              <Insta className="h-8 w-8" />
+            </a>
+            <a
+              href="/facebook"
+              className="flex items-center justify-center bg-natural-50 h-10 w-10 rounded-full"
+            >
+              <Facebook className="h-8 w-8" />
+            </a>
+            <a
+              href="/linkedin"
+              className="flex items-center justify-center bg-natural-50 h-10 w-10 rounded-full"
+            >
+              <Linkedin className="h-8 w-8" />
+            </a>
+            <a
+              href="/whatsapp"
+              className="flex items-center justify-center bg-natural-50 h-10 w-10 rounded-full"
+            >
+              <Whatsapp className="h-8 w-8" />
+            </a>
+            <a
+              href="/youtube"
+              className="flex items-center justify-center bg-natural-50 h-10 w-10 rounded-full"
+            >
+              <Youtube className="h-8 w-8" />
+            </a>
+          </div>
```

Comment:
For simplicity, we can move this into a seperate component `<Social />` (in the same file!). All items has the same styles, we can make a list and loop over them.